docs/source/tutorial/11-subscriptions.mdx

```diff
@@ -110,11 +108,13 @@ In `MainActivity`, register your subscription and start listening to events usin
     }
 ```
 
+> NOTE: You may need to `import kotlinx.coroutines.flow.collect` if you see an error about internal use for `collect`. 
```

Comment:
👍 for some reason AS often has problem auto-importing this one, I have no idea why!

Reply:
I wonder if there's another `collect` that might get it confused, that's supposed to be internal-only, and so AS is like OH NOES